### PR TITLE
Feature/sidechain connector service tests+simple ganache/ethereum simulation factory

### PR DESCRIPTION
**Pull request by jlevison**

### CODE REVIEW

File: projects/services/sidechain-connector-service-typescript/test/ethereum-driver.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+
+  function getValues() public view returns (uint256 intValue, string stringValue) {
+    intValue = item.intValue;
+    stringValue = item.stringValue;
+  }
+}
+`;
+
+export abstract class EthereumSim {
```
[the remark this answers is not in the document]
Comment (jlevison) [reply projects/services/sidechain-connector-service-typescript/test/ethereum-driver.ts]: sure

File: projects/services/sidechain-connector-service-typescript/test/ethereum-driver.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        const account = (await web3.eth.getAccounts())[0];
+        const contractInstance = await <Contract>tx.send({
+            from: account,
+            gas: await tx.estimateGas()
+        });
+
+        this.storedInt = intValue;
+        this.storedString = stringValue;
+
+        return Promise.resolve<string>(contractInstance.options.address);
```
[the remark this answers is not in the document]
Comment (jlevison) [reply projects/services/sidechain-connector-service-typescript/test/ethereum-driver.ts]: k, was not sure myself, changing it

File: projects/services/sidechain-connector-service-typescript/.yarnrc [hunk 3]
```diff
@@ -1,4 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (netoneko) [inline projects/services/sidechain-connector-service-typescript/.yarnrc]: Please move it back. It's there for a reason.

Comment (jlevison) [reply projects/services/sidechain-connector-service-typescript/.yarnrc]: yeah, i agree with @idandash its wrong, what are we missing?

Comment (netoneko) [reply projects/services/sidechain-connector-service-typescript/.yarnrc]: It's there because it cuts the build time and the size of the image.

**Pull request merged**